### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by quazi-broad**
I think the vaultVersion should be "kv1" - at least based on the experience Dan had.
The secrets-manager installation suddenly stopped being able to read paths from Vault but Dan was able to resolve this by switching the vaultVersion value from "kv2" to "kv1".
DevOps confirms that everything under secret/* is a v1 engine.

### CODE REVIEW

**Comment by smark88** [comment]
You can over ride it in a values file you supply the default is the industry standard.

**Comment by danxmoran** [comment]
It's industry standard but our own Vault instance doesn't support it (confirmed by Greg), no clue how it was ever working for us.

File: charts/install-secrets-manager/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The following tables lists the configurable parameters of the install-secrets-ma
...
 | `vaultLocation`                        | URI path to your vault instance registry                                                                                                                                              | `nil`                                                         |
-| `vaultVersion`                        | vault key value version registry                                                                                                                                              | `kv2`                                                         |
+| `vaultVersion`                        | vault key value version registry                                                                                                                                              | `kv1`                                                         |
```
Comment (danxmoran) [inline charts/install-secrets-manager/README.md]: If we decide to go through with changing the default, we'll need to update the `values.yaml` too.

Comment (smark88) [reply charts/install-secrets-manager/README.md]: I always set kv1

**Comment by quazi-broad** [comment]
The values file can be overridden.

**Pull request closed without merging**